<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consultarNegos</class>
 <widget class="QDialog" name="consultarNegos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboOrigenFilter">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDestinoFilter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFechaFilter">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>23</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFechaFilter">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>23</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="mostrar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelOrigen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Origen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOrigen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="placeholderText">
     <string>Origen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDestino">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="placeholderText">
     <string>Destino</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDestino">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Destino&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFecha">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha vuelo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGuion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFechaDis">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha dis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlazas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Plazas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="plazas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="porcentaje">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPorcDis">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;% Dis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateVuelo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>38</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeVuelo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDis">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>38</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelOrigenFilter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>23</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Origen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDestinoFilter">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>23</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Destino&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaNegos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>800</width>
     <height>350</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QCheckBox" name="checkFecha">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>23</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboOrigenFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>consultarNegos</receiver>
   <slot>selectedOrigen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboDestinoFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>consultarNegos</receiver>
   <slot>selectedDestino()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFechaFilter</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>consultarNegos</receiver>
   <slot>selectedFecha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFecha</sender>
   <signal>clicked()</signal>
   <receiver>consultarNegos</receiver>
   <slot>selectedFecha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablaNegos</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>consultarNegos</receiver>
   <slot>showLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablaNegos</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>consultarNegos</receiver>
   <slot>showLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showNegos()</slot>
  <slot>activarFecha()</slot>
  <slot>showLabel()</slot>
  <slot>selectedOrigen()</slot>
  <slot>selectedDestino()</slot>
  <slot>selectedFecha()</slot>
 </slots>
</ui>
